build/MSBuildExtensions.targets

```diff
@@ -158,7 +158,7 @@
       <ImplicitPackageVariable Include="Microsoft.NETCore.App"
                                TargetFrameworkVersion="2.1"
                                DefaultVersion="2.1.0"
-                               LatestVersion="2.1.2" />
+                               LatestVersion="2.1.5" />
```

Comment:
2.1.5 is not out yet.

Reply:
Apparently it's available on the feeds we're using to build 2.2.  Will 2.1.5 be out by the time 2.2 ships?